src/converters/lintConfigs/rules/meta.test.ts

```diff
@@ -0,0 +1,84 @@
+import { promises as fs } from "fs";
+import { resolve, join } from "path";
+
+describe("@typescript-eslint rules that require another rule be disabled", () => {
```

Comment:
Err, I don't think this test needs to be written? A lot of the rules (e.g. `default-param-last`) weren't TSLint rulees and thus aren't relevant to this project. The rest are already tested in their own files.

Reply:
I written these so I could find the buggy rules and thought if I already written it, it can stay.
If/when a new conversion is added which forgets to disable a rule from here a test will break.

But if you want I will delete it.